application/templates/static_site/measure.html

```diff
@@ -182,45 +183,61 @@ <h2 class="govuk-heading-l">There is a new version of this page</h2>
-            Things you need to know
-          </span>
-        </summary>
-        <div class="govuk-details__text">
-            {{ measure_version.need_to_know | render_markdown  }}
-        </div>
-      </details>
+
+        {{ govukDetails(
+          summaryText="Things you need to know",
+          html=measure_version.need_to_know | render_markdown,
+          classes="govuk-!-margin-bottom-0",
+          id="things-you-need-to-know",
+          attributes={
+            "data-on":"click",
+            "data-event-category":"Disclosure",
+            "data-event-action":"Opened",
+            "data-event-label":"Things you need to know"
+          }
+        ) }}
+
       {% endif %}
 
       {% if measure_version.measure_summary %}
-      <details class="govuk-details govuk-!-margin-bottom-0" data-on="click" data-event-category="Disclosure" data-event-action="Opened" data-event-label="What the data measures" class="normalize">
-        <summary class="govuk-details__summary">
-          <span id="what-the-data-measures" class="govuk-details__summary-text">
-            What the data measures
-          </span>
-        </summary>
-        <div class="govuk-details__text" itemprop="description">
-            {{ measure_version.measure_summary | render_markdown  }}
-        </div>
-      </details>
+
+        {{ govukDetails(
+          summaryText="What the data measures",
+          html=measure_version.measure_summary | render_markdown,
+          classes="govuk-!-margin-bottom-0",
+          id="what-the-data-measures",
+          attributes={
+            "data-on":"click",
+            "data-event-category":"Disclosure",
+            "data-event-action":"Opened",
+            "data-event-label":"What the data measures"
+          }
+        ) }}
+
       {% endif %}
 
       {% if measure_version.ethnicity_definition_summary %}
-      <details class="govuk-details" data-on="click" data-event-category="Disclosure" data-event-action="Opened" data-event-label="The ethnic categories used in this data" class="normalize">
-        <summary class="govuk-details__summary">
-          <span id="the-ethnic-categories-used-in-this-data" class="govuk-details__summary-text">
-            The ethnic categories used in this data
-          </span>
-        </summary>
-        <div class="govuk-details__text">
-            {{ measure_version.ethnicity_definition_summary | render_markdown  }}
+
+        {% set ethnicCategoriesHTML %}
+          {{ measure_version.ethnicity_definition_summary | render_markdown }}
           <p class="govuk-body"><a class="govuk-link" href="{{ url_for('static_site.ethnicity_in_the_uk') }}">Ethnic groups and how data on ethnicity is collected</a></p>
-        </div>
-      </details>
+        {% endset %}
+
+
+        {{ govukDetails(
+          summaryText="The ethnic categories used in this data",
+          html=ethnicCategoriesHTML,
+          classes="",
```

Comment:
Hmmm - does this need to be specified at all if the value is blank? Does it override some default that I'm not seeing, or is this redundant?

Reply:
@samuelhwilliams good spot, it's redundant, will remove.